src/firmware/app/primitives/primitive_manager.c

```
@@ -200,11 +196,20 @@ void app_primitive_manager_startNewPrimitive(PrimitiveManager_t *manager,
         }
         default:
         {
-            app_primitive_manager_makeRobotSafe(manager, world);
-            assert(false);
+            // the estop case is handled here
+            app_primitive_makeRobotSafe(world);
+            return;
```

Not against the change, just curious why `return` over `assert(false)`? The assert would crash and burn, while the return fails silently.